source/source_integration_test.go

```diff
@@ -50,12 +60,12 @@ func TestSource_Read_NoTable(t *testing.T) {
 	is.NoErr(err)
 
 	err = src.Open(ctx, nil)
-	is.True(err != nil)
+	is.True(strings.Contains(err.Error(), "table or view does not exist"))
```

Comment:
Maybe mention table name in the error as well?

Reply:
The full error is:
```
new iterator: init snapshot iterator: initialize snapshot: create snapshot: ORA-00942: table or view does not exist
```

As you can see we got this error when we try to create a snapshot of a table.
But it can be another error during the execution.

Do we need to check the error and in the case of this error add the table name?